docs/api_v0.yml

```diff
@@ -691,7 +691,7 @@ components:
     ListingCreate:
       type: object
       properties:
-        classified_listing:
```

Comment:
this and the rest of changes are going to break the API though, I think we should either accept both versions of leave it there as is

Reply:
I'm always a bit unsure about this: we have now on numerous occasions broken the API and justified this by citing the v0 version number. Since we separated internal and external API, this would only be used by tools built by others, do we know if they exist/how many requests they make?

Reply:
@rhymes and I discussed this on Slack and decided to leave this as-is for now and wait for TheGreatAPIRefactor™ before we release the v1 API.